content/en/docs/contribute/_index.md

```diff
@@ -16,7 +16,7 @@ This website is maintained by [Kubernetes SIG Docs](/docs/contribute/#get-involv
 
 Kubernetes documentation contributors:
 
-- Improve existing content
+- [Improve existing content](https://github.com/kubernetes/website/contribute)
```

Comment:
I think this would work better in the “Your first contribution” section (that starts on line 44). What do you think @Heena-PyCloud ?

My idea is to tell the reader where to find inspiration for their first pull request.

Reply:
@sftim Hi Tim,
That sounds good. I have made the change as per your suggestion.